grakn-test/src/test/java/ai/grakn/test/GraknTestEnv.java

```diff
@@ -146,10 +146,12 @@ static String randomKeyspace(){
     }
 
     public static void hideLogs() {
-        ((Logger) org.slf4j.LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME)).setLevel(Level.OFF);
-        ((Logger) org.slf4j.LoggerFactory.getLogger(GraknTestEnv.class)).setLevel(Level.DEBUG);
-        ((Logger) org.slf4j.LoggerFactory.getLogger(BackgroundTaskTestUtils.class)).setLevel(Level.DEBUG);
+        ((Logger) org.slf4j.LoggerFactory.getLogger(ROOT_LOGGER_NAME)).setLevel(Level.ERROR);
```

Comment:
I think we shouldn't be changing the log level programmatically at all - we should be using main and test `logback.xml` files to set these levels.